spec/Section 2 -- Language.md

```diff
@@ -587,13 +587,13 @@ FragmentName : Name but not `on`
 
 Fragments are the primary unit of composition in GraphQL.
 
-Fragments allow for the reuse of common repeated selections of fields, reducing
-duplicated text in the document. Inline Fragments can be used directly within a
-selection to condition upon a type condition when querying against an interface
-or union.
+Fragments allow for the definition of selection sets that are colocated with the 
+logic that requires those selections, making it easier to add and remove selections 
+as needed.
```

Comment:
Tentative suggestion that keeps the concept of "colocation" out of the spec for the time being.

```suggestion
Fragments allow grouping selections that belong together, making it easier to scope 
data accesses, add and remove selections as needed.
```